### PR TITLE
PR for To do List

### PR DESCRIPTION
**Pull request by smcommits**
# Description

In this PR, We have implemented a simple to do list application. The goal  was to use javascript to create an application where a user can create new projects, add tasks for each project, edit the project and tasks, and delete the tasks. 

## Page Contents 
- Project List 
- Task List
- Controll Buttons
- Create and edit form 

## Javascript
We have used the ES6 version of Javascript. We have used the MVC software design to divide the code and responsibility. 

## Technologies Used
- Webpack for bundling the code 
- NPM for dependencies and script handling
- ESLINT for lining Javascript
- Sylelint for lining CSS

### CODE REVIEW

**Review by Forison — CHANGES_REQUESTED** [review]
## Status::Change required❌

# 🤗 team, 

#### You have done a great job on this one, I Iike the way the project has been organized, the code base is very clean and readable. The UI is also amazing. Here is one requirement we have to address and other optional requirement that could even make the app better😇 .

* [ ] Let us make sure the necessary run common commands like `build`, `development` should be in `package.json` inside \`scripts'. Here is a basic example ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/todo_list/scripts_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622602/))
- Let add this ``"build": "webpack --mode production",`` to our script.

**[Optional]** 
* [ ] The forms used to create a project or a todo should have validations, you should not be able to create an empty project/todo, you can simply add the required attribute to your input fields.
* [ ] Currently we can create duplicate projects name, I think porject it would be better if project is unique, that is just my view.

#### That is it, kindly make the changes and submit a re-review

#### Happy coding🙂
Addo Forison

File: package.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{
+  "name": "task_manager",
+  "version": "1.0.0",
+  "description": "This is simple task manager that you can create to-do lists for projects.",
+  "main": "index.js",
+  "scripts": {
+    "start:dev": "webpack serve --open",
```
Comment (Forison) [inline package.json:7]: `"build": "webpack --mode production",` this would help to run the `npm run build command` for deployment

**Review by Forison — CHANGES_REQUESTED** [review]
## Status::Change Required❌ 
## Somethings seems to be wrong with the update you made, The application is not longer responding, even the default project is not showing. I get this in my console🙂 both in development and production.
Screenshot from 2021-03-17 15-09-39

Let us address this and submit another review.

Regards
Addo Forison

**Comment by smcommits** [comment]
@Forison Hey! Thanks for the feedback. I suspect that there is an issue with your localStorage as the application runs fine on multiple machines I have tested. Can you please try to clean the localStorage and run the application again? Please feel free to let me know if the issue is integral to the application.

**Review by NtwaliHeritier — APPROVED** [review]
### **STATUS: APPROVED✔✔✔**

Great work making the required changes👏👏👏
You can merge and move on👌

Happy coding💻💻

**Pull request merged**